src/rgw/rgw_rest_s3.cc

```diff
@@ -408,14 +417,28 @@ void RGWGetUsage_ObjStore_S3::send_response()
        formatter->dump_int("SuccessfulOps", total_usage.successful_ops);
        formatter->close_section(); // total 
        formatter->close_section(); // user
-     } 
-     formatter->dump_int("TotalBytes", header.stats.total_bytes);
```

Comment:
@theanalyst 
If really someone else used this api ，I agree to keep it as before。
I will change it back